ChangeLog.rst

```diff
@@ -9,6 +9,94 @@ Unreleased
   releasing next version, the "Unreleased" header will be replaced with
   appropriate version header and this help text will be removed.
 
+Version 2.0.0
+-------------
```

Comment:
Don't put the 2.0.0 header just yet. See 81fbe29feb037fb8dc32c252fac097c8fbd17ccd and e3dd81627de41648de90833c3555c3a65ac2b991.